lib/diagnostic.rb

```diff
@@ -133,19 +159,23 @@ def initialize
   end
 
   def punch
-    @moves << 'punch'
-    @damage += 5
-    @damage *= multiplier
+    self.moves << 'punch'
```

Comment:
This is unnecessary unless you specifically want to use the `self.variable` or `self.variable=` methods.